src/utils/omit.ts

```diff
@@ -0,0 +1,4 @@
+// Based on https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
Omit is part of typescript 3.5+, I don't think we need this.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type

Reply:
Updated